test/concat-test.js

```diff
@@ -22,26 +22,32 @@ describe('concat', function () {
 		assert.strictEqual(p2, p1.concat(p2))
 	})
 
-	it('should return earlier future', () => {
+	it('should behave like earlier future', () => {
+		const expected = {}
+		const p = delay(1, expected).concat(delay(10))
+		return assertSame(p, fulfill(expected))
+	})
+
+	it('should behave like other earlier future', () => {
 		const expected = {}
 		const p = delay(10).concat(delay(1, expected))
 		return assertSame(p, fulfill(expected))
 	})
 
-	it('should behave like fulfilled', () => {
+	it('should return other with fulfilled', () => {
```

Comment:
I think the "be identity with" terminology will work here too (as it does below with the `never` test).